### ISSUE TITLE
Disable certain elements in overlay tool

### ISSUE THREAD

**Issue by FranzAr** [fpv-wtf/wtfos-configurator#359]
I don't know if this is possible but I'd really love to see an option in the online overlay tool to disable certain OSD elements like GPS position or altitude for some reasons ;-)

**Comment by stylesuxx** [comment]
Technically, I think it should be possible - and removing GPS data post flight might make sense.
@Knifa - any input on this?

**Comment by j005u** [comment]
There's no eloquent "please draw GPS coords here" in the .osd we could simply ignore.

The problem is the entire screen is a 60x22 character grid, where the FC simply draws icons + text arbitarily.

With that said it's probably possible to use some ugly logic to detect the appropriate icon glyph drawn before the actual gps coords and then also ignore the next n characters on the grid.

It's definitely specific to the FC at hand though, I have no idea how we'd do this without making it an ugly special case hack. Allow advanced users to filter out occurrences of glyph x + y characters after it on the same line? How do you configure this so it's sane? Or only support BF?

**Comment by j005u** [comment]
Another thought occurred:
Just allow the user to mask out a specific area on the grid. However this wouldn't play all that nice with end-of-flight screens.

**Comment by stylesuxx** [comment]
@j005u - Yeah, you are right - this probably is overkill.

We can currently render the OSD without merging it with the video itself - this would allow the user to remove whichever sections he does not want to have in post and then merge OSD and "censored" video by himself.

**Comment by FranzAr** [comment]
> Another thought occurred: Just allow the user to mask out a specific area on the grid. However this wouldn't play all that nice with end-of-flight screens.

Yeah, I think that could be a possible solution. Stat screen could be cut off or masked by user if he wants to.



> @j005u - Yeah, you are right - this probably is overkill.
> 
> We can currently render the OSD without merging it with the video itself - this would allow the user to remove whichever sections he does not want to have in post and then merge OSD and "censored" video by himself.

Yes, that's possible but another time eating step to do. 

It just would be nice to do that all at once. But if you think it's overkill I'm fine with that too. Just an idea that came across my mind.

**Comment by stylesuxx** [comment]
I'll keep this issue open to see if other users might show interest in such a feature, but generally speaking I think we provide all the tools to do this with minimal added effort for the user by separately rendering the OSD and the user having the option to censor what he wants in post.

**Comment by ThisIsBrutus** [comment]
Very interested in this one way or the other. Yes, blurring it out in post is easy, but saving that trip to davinci and another render would be awesome for those videos where you just wanna upload the whole thing (but without incriminating yourself too much)
